arches/app/views/user.py

```diff
@@ -174,11 +178,12 @@ def post(self, request):
                         + settings.APP_NAME
                         + " profile was just changed.  If this was unexpected, please contact your "
                         + settings.APP_NAME
-                        + " administrator at %s." % (admin_info)
+                        + " administrator"
+                        + (" at %s." % (admin_info) if (admin_info != "") else ".")
```

Comment:
@RobOatesHistoricEngland, could you change this to use `format` instead of the old string formatting? We are trying to move to more consistent string formatting syntax throughout Arches. I think the line would look like: ```(" at {0}".format(admin_info) if admin_info != "" else ".")```